code/modules/mob/living/carbon/xenomorph/castes/hunter/abilities_hunter.dm

```diff
@@ -13,15 +13,16 @@
 	var/stealth = FALSE
 	var/can_sneak_attack = FALSE
 	var/stealth_alpha_multiplier = 1
+	var/last_run_intent
```

Comment:
why is this named intent if it is a cooldown, and why does it not use the cooldown defines

Reply:
It's not a true cooldown as it does not prevent sneak attacks or stealth from working completely, it just reduces their functionality if not enough time has elapsed since the Hunter last moved while on run intent.

Reply:
If they behave like cooldowns so they should use them